vhpc_toolkit/get_objs.py

```diff
@@ -51,6 +51,16 @@ def get_container_view(self, vimtype):
         )
         return container.view
 
+    def get_objs(self, vimtype, name):
+        objs = []
+        for c in self.get_container_view(vimtype):
```

Comment:
Why we need to use the container view?
Change `c` to a meaningful name, e.g., `container`